extra-utils/xss-lock/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=xss-lock
+PKGSEC=utils
+PKGDEP="glib xcb-util"
+PKGDES="A tool helps use external locker as X screen saver"
```

Comment:
Please rephrase this (awkward sentence, misspelled XScreenSaver). I'd suggest:

"A tool for using external lockers as XScreenSaver replacements"